src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs

```diff
@@ -133,6 +133,13 @@ public string[] Formats
         public string? Format => TryGetValue("x-ms-format", out object? value) ? value?.ToString() : null;
     }
 
+    internal static class XMsFormat
+    {
+        public const string ArmId = "armId";
```

Comment:
So why not just simple value `resourceId` like `resourceType` in the next line? Is this avoiding some conflicts with something else?

Reply:
You mean name it resourceId instead of armId? I thought armId is more commonly used in terms of what its format should look like.